types/context.go

```diff
@@ -66,15 +66,13 @@ type Context struct {
 type Request = Context
 
 // Read-only accessors
-func (c Context) Context() context.Context          { return c.baseCtx }
-func (c Context) MultiStore() storetypes.MultiStore { return c.ms }
-func (c Context) BlockHeight() int64                { return c.header.Height }
-func (c Context) BlockTime() time.Time              { return c.header.Time }
-func (c Context) ChainID() string                   { return c.chainID }
-func (c Context) TxBytes() []byte                   { return c.txBytes }
-func (c Context) Logger() log.Logger                { return c.logger }
-
-// Deprecated: use Cometinfo.GetLastCommit().Votes() instead, will be removed after 0.51
```

Comment:
I think the comment should not be in this PR

Reply:
adjusted the commented but it is correct to be here. the other pr was merged